Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -34,7 +37,54 @@ public static class TextTable
         /// </summary>
         public static string Build(string message, int padding)
         {
-            return "";
+            if (message.Length == 0 || message == null) return "";
+
+            var textTable = new StringBuilder();
+            string[] seperatedMessage = message.Split(new[] { Environment.NewLine }, StringSplitOptions.None);
+
+            // Find the max length of the longest word in the message
+            int maxCharacters = 0;
+            foreach (var word in seperatedMessage)
+            {
+                if (word.Length > maxCharacters)
+                {
+                    maxCharacters = word.Length;
+                }
+            }
+
+            // Make-up of number of characters
+            int maxWidth = maxCharacters + (padding * 2);
+            string minus = new string('-', maxWidth);
+            string emptyLine = new string(' ', maxWidth);
+            string emptySpace = new string(' ', padding);
+
+            // TopLine
+            textTable.Append($"+{minus}+\r\n");
+
+            // NewLine if padding
+            for (int i = 0; i < padding; i++)
```

Comment:
Fyi sometimes loops like this can be avoided by re-framing things just a little like so:

`textTable.Append(string.Join(Environment.NewLine, Enumerable.Repeat($"|{emptyLine}|", padding)));`

Reply:
I had to add \r\n because Environment.NewLine didn't do anything?